gamemode/modules/base/sv_data.lua

```diff
@@ -236,6 +236,16 @@ function DarkRP.retrieveRPNames(name, callback)
     end)
 end
 
+function DarkRP.offlinePlayerData(steamid, callback, failed)
+    local split = string.Explode(":",steamid:gsub("STEAM_",""))
+    if (not split[1] or not split[2] or not split[3]) then failed() return end
+    local uniqueid = util.CRC("gm_STEAM_" .. split[1] .. ":" .. split[2] .. ":" .. split[3] .. "_gm")
```

Comment:
@AMD-NICK ew how did I not see that

@FPtje from previous experiences it seems that you don't like Issues so if something fucks up and there's an error then 👌